### PR TITLE
Add Notion Detection

### PR DESCRIPTION
**Pull request by ysm0622**
website:
- https://notion.so

examples: 
- https://www.notion.so/Notion-Community-04f306fbf59a413fae15f42e2a1ab029
- https://www.notion.so/sablecard/Sable-is-Hiring-472678e95a374c9cb5040252409a0714
- https://www.notion.so/useparagon/Paragon-is-Hiring-ab5739dac1c4490db920775088c618dd
- https://socarcorp.notion.site/socarcorp/d458b6b77a2243fb873d1ac800c321f7
- https://www.notion.so/Work-at-Lemonbase-7f055b51bc334a618f04903897aea351

### CODE REVIEW

File: src/technologies/n.json [hunk 1]
```diff
@@ -721,6 +721,22 @@
     "scriptSrc": "connect\\.nosto\\.\\w+/",
     "website": "https://www.nosto.com"
   },
+  "Notion": {
+    "cats": [
+      51
+    ],
+    "description": "The all-in-one workspace — for notes, tasks, wikis, and databases.",
```
Comment (nurbek91) [inline src/technologies/n.json:728]: "description": "Notion is a collaboration platform with modified Markdown support that integrates kanban boards, tasks, wikis, and database.",

File: src/technologies/n.json [hunk 2]
```diff
@@ -721,6 +721,22 @@
...
+    "icon": "Notion.svg",
```
Comment (nurbek91) [inline src/technologies/n.json:729]: 32x32
[Notion.svg.zip](https://github.com/wappalyzer/wappalyzer/files/8985263/Notion.svg.zip)

**Review by nurbek91 — CHANGES_REQUESTED** [review]
Hey mate, thanks for PR!!
Needs minor updates to bring it to perfection

additional examples:
https://xuebin.me/
https://slahser.com/

**Comment by ysm0622** [comment]
@nurbek91 Thanks! [a3a00b8](https://github.com/wappalyzer/wappalyzer/pull/6415/commits/a3a00b86025245ed717d455991182efbbc44fa04)

**Review by nurbek91 — APPROVED** [review]
thanks mate!!!

**Pull request merged**